tools/configure.sh

```diff
@@ -296,7 +296,7 @@ done
 
 if [ "X${defappdir}" = "Xy" ]; then
   # In-place edit can mess up permissions on Windows
-  # sed -i -e "/^CONFIG_APPS_DIR/d" "${dest_config}"
```

Comment:
Commented out seems like we can just drop this

Reply:
maybe this instance can be replaced with kconfig-tweak too

Reply:
Looks like the commented section can be removed and the next line updated. It was coping to a temporary file editing via sed and moving back.

Reply:
The change is same as before so, we can't use kconfig-tweak here?

Reply:
> maybe this instance can be replaced with kconfig-tweak too

Looks like the commented section can be removed and the next line updated. It was coping to a temporary file editing via sed and moving back.